src/IIIFPresentation/API/Features/Storage/Requests/UpsertCollection.cs

```diff
@@ -112,6 +125,11 @@ public async Task<ModifyEntityResult<PresentationCollection, ModifyCollectionTyp
                 return ModifyEntityResult<PresentationCollection, ModifyCollectionType>.Failure(
                     "ETag does not match", ModifyCollectionType.ETagNotMatched, WriteResult.PreConditionFailed);
             }
+            
+            if (isStorageCollection && !databaseCollection.IsStorageCollection)
+            {
```

Comment:
Worth logging here? `Customer X attempt to move collection Y` to help with any potential customer questions. 

This prevents iiif -> storage collection, should reverse be allowed?

Reply:
From a technical standpoint, I think so? The main issue with moving between IIIF to storage is what to do with the S3 JSON, whereas going the other way this isn't an issue - though there might need to be a discussion about whether this is a good user experience

Reply:
after conversation: wrinkle in this is that storage collections can have child items, but IIIF collections can't - as such better to just disallow all conversions